---
[v1] arm64: dts: mt7988: add led config for aqr 10gphy

Add LED configuration for AQR 10G PHY with three LED nodes per PHY
for link status indication. Enables netdev trigger for network activity LEDs.
---
v1: Add LED properties for our AQR113C RFB.
---

From 08ffe0bbe320a8ce67b2be4238c42f29018d7e74 Mon Sep 17 00:00:00 2001
From: Mason Chang <mason-cw.chang@mediatek.com>
Date: Wed, 28 Jan 2026 11:13:33 +0800
Subject: [PATCH] arm64: dts: mt7988: add led config for aqr 10gphy

---
 .../dts/mediatek/mt7988a-rfb-eth1-aqr.dtso    | 23 +++++++++++++++++++
 .../dts/mediatek/mt7988a-rfb-eth2-aqr.dtso    | 23 +++++++++++++++++++
 2 files changed, 46 insertions(+)

--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-aqr.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-aqr.dtso
@@ -28,6 +28,29 @@
 				eee-broken-5gt;
 				eee-broken-2_5gt;
 				eee-broken-1000t;
+
+				leds {
+					#address-cells = <1>;
+					#size-cells = <0>;
+
+					phy0_led0: aqr_phy0_led0@0 {
+						reg = <0>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+
+					phy0_led1: aqr_phy0_led1@1 {
+						reg = <1>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+
+					phy0_led2: aqr_phy0_led2@2 {
+						reg = <2>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+				};
 			};
 		};
 	};
--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-aqr.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-aqr.dtso
@@ -28,6 +28,29 @@
 				eee-broken-5gt;
 				eee-broken-2_5gt;
 				eee-broken-1000t;
+
+				leds {
+					#address-cells = <1>;
+					#size-cells = <0>;
+
+					phy8_led0: aqr_phy8_led0@0 {
+						reg = <0>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+
+					phy8_led1: aqr_phy8_led1@1 {
+						reg = <1>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+
+					phy8_led2: aqr_phy8_led2@2 {
+						reg = <2>;
+						linux,default-trigger = "netdev";
+						status = "okay";
+					};
+				};
 			};
 		};
 	};
